new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/WebLogicImageTool.java

```diff
@@ -55,7 +59,21 @@ public boolean updateImage() {
     } 
 
     try {
+      // check if the WIT binary exists, throws FileNotFoundException if the file does not exist
```

Comment:
didn't you literally just download it and check it was there a few lines ago? 
do we need another check?

Reply:
Although we have java code doing the download and unzip for now, the plan is to use a script to download all installers upfront before the Java test starts. We'll probably need this check then.  I could remove this check now,  and add it back when/if we get rid of the java Installer code.

Reply:
removed the check